packages/boilerplate-generator-tests/web.cordova-tests.js

```diff
@@ -60,9 +59,7 @@ Tinytest.addAsync(
   async function (test) {
     const newHtml = await generateHTMLForArch('web.cordova', false);
 
-    _.templateSettings = {
```

Comment:
This being removed makes me feel that we are missing here something even though the tests are passing.

Reply:
It's used within tests and doesn't affect the module functionality. If the tests fail, I'll re-look into it.